tests/testChannel.cpp

```diff
@@ -0,0 +1,189 @@
+#include <catch2/catch_approx.hpp>
+
+#include "Channel.hpp"
+#include "testUtils.hpp"
+
+TEST_CASE("Test Channel Construction and Basic Operations", "[channel]")
+{
+  SECTION("Default constructor parameters")
+  {
+    Channel ch("test_channel", "test_group", 1, 2, 3);
+    REQUIRE(ch.getName() == "test_channel");
+    REQUIRE(ch.getGroupName() == "test_group");
+    REQUIRE(ch.getGroupIndex() == 1);
+    REQUIRE(ch.getLocalIndex() == 2);
+    REQUIRE(ch.getGlobalIndex() == 3);
+    REQUIRE(ch.getConversion()
+            == Catch::Approx(0.05e-6f).epsilon(
+                0.001));  // bitVolts/conversion = 0.05/1e6
+    REQUIRE(ch.getSamplingRate() == Catch::Approx(30000.f).epsilon(0.001));
+    REQUIRE(ch.getBitVolts() == Catch::Approx(0.05f).epsilon(0.001));
+    const auto& pos = ch.getPosition();
+    REQUIRE(pos[0] == Catch::Approx(0.f).epsilon(0.001));
+    REQUIRE(pos[1] == Catch::Approx(0.f).epsilon(0.001));
+    REQUIRE(pos[2] == Catch::Approx(0.f).epsilon(0.001));
+    REQUIRE(ch.getComments() == "no comments");
+  }
+
+  SECTION("Custom constructor parameters")
+  {
+    std::array<float, 3> pos = {1.f, 2.f, 3.f};
+    Channel ch("test_channel",
+               "test_group",
+               1,
+               2,
+               3,
+               2e6f,  // custom conversion
+               44100.f,  // custom sampling rate
+               0.1f,  // custom bitVolts
+               pos,  // custom position
+               "test comment"  // custom comment
+    );
+
+    REQUIRE(ch.getName() == "test_channel");
+    REQUIRE(ch.getGroupName() == "test_group");
+    REQUIRE(ch.getGroupIndex() == 1);
+    REQUIRE(ch.getLocalIndex() == 2);
+    REQUIRE(ch.getGlobalIndex() == 3);
+    REQUIRE(ch.getConversion() == Catch::Approx(0.1f / 2e6f).epsilon(0.001));
+    REQUIRE(ch.getSamplingRate() == Catch::Approx(44100.f).epsilon(0.001));
+    REQUIRE(ch.getBitVolts() == Catch::Approx(0.1f).epsilon(0.001));
+    const auto& actualPos = ch.getPosition();
+    REQUIRE(actualPos[0] == Catch::Approx(pos[0]).epsilon(0.001));
+    REQUIRE(actualPos[1] == Catch::Approx(pos[1]).epsilon(0.001));
+    REQUIRE(actualPos[2] == Catch::Approx(pos[2]).epsilon(0.001));
+    REQUIRE(ch.getComments() == "test comment");
```

Comment:
I think we can use the `Catch::Matchers::RangeEquals` with an approximate comparator to simplify our comparison of vectors/arrays of floats throughout these tests. 

```suggestion
    auto approxComparator = [](float a, float b) { 
        return a == Catch::Approx(b).epsilon(0.001);   # define once or in testUtils
    };
    REQUIRE_THAT(actualPos, Catch::Matchers::RangeEquals(pos, approxComparator));
```